scipy/sparse/__init__.py

```diff
@@ -84,53 +77,77 @@
-   rand - Random values in a given shape matrix (old interface)
+   kron - Kronecker product of two sparse arrays
+   kronsum - Kronecker sum of sparse arrays
+   block_diag - Build a block diagonal sparse array
+   tril - Lower triangular portion of a sparse array
+   triu - Upper triangular portion of a sparse array
+   hstack - Stack sparse arrays horizontally (column wise)
+   vstack - Stack sparse arrays vertically (row wise)
 
-Building larger structures from smaller (array or matrix)
+Sparse tools
+------------
 
 .. autosummary::
    :toctree: generated/
 
-   kron - kronecker product of two sparse matrices
-   kronsum - kronecker sum of sparse matrices
-   block_diag - Build a block diagonal sparse matrix
-   tril - Lower triangular portion of a matrix in sparse format
-   triu - Upper triangular portion of a matrix in sparse format
-   hstack - Stack sparse matrices horizontally (column wise)
-   vstack - Stack sparse matrices vertically (row wise)
+   save_npz - Save a sparse array to a file using ``.npz`` format.
+   load_npz - Load a sparse array from a file using ``.npz`` format.
+   find - Return the indices and values of the nonzero elements
 
-Save and load sparse matrices:
+Identifying sparse arrays
+-------------------------
+
+- use ``isinstance(A, sp.sparse.sparray)`` to check whether an array or matrix.
+- use ``A.format == 'csr'`` to check the sparse format
 
 .. autosummary::
    :toctree: generated/
 
-   save_npz - Save a sparse matrix/array to a file using ``.npz`` format.
-   load_npz - Load a sparse matrix/array from a file using ``.npz`` format.
+   issparse - Check is the argument is a sparse object (array or matrix).
+
 
-Sparse tools:
+Sparse matrix classes
+=====================
 
 .. autosummary::
    :toctree: generated/
 
-   find
+   bsr_matrix - Block Sparse Row matrix
+   coo_matrix - A sparse matrix in COOrdinate format
+   csc_matrix - Compressed Sparse Column matrix
+   csr_matrix - Compressed Sparse Row matrix
+   dia_matrix - Sparse matrix with DIAgonal storage
+   dok_matrix - Dictionary Of Keys based sparse matrix
+   lil_matrix - Row-based list of lists sparse matrix
+   spmatrix - Sparse matrix base class
 
-Identifying sparse arrays:
+Building sparse matrices
+------------------------
 
-- use ``isinstance(A, sp.sparse.sparray)`` to check whether an array or matrix.
-- use ``A.format == 'csr'`` to check the sparse format
+.. autosummary::
+   :toctree: generated/
 
-Identifying sparse matrices:
+   eye - Sparse MxN matrix whose k-th diagonal is all ones
+   identity - Identity matrix in sparse matrix format
+   diags - Return a sparse matrix from diagonals
+   spdiags - Return a sparse matrix from diagonals
+   bmat - Build a sparse matrix from sparse sub-blocks
+   random - Random values in a given shape matrix
+   rand - Random values in a given shape matrix (old interface)
+
+**Stacking matrices use the same functions as for** `stacking arrays`_
```

Comment:
minor reST thing: we normally use dashes for reST labels, not spaces (a bit easier to work with, just like spaces in paths)

Reply:
Thanks for this tip. Makes sense to me.